metarParser-services/src/test/java/io/github/mivek/service/WeatherCategoryServiceTest.java

```diff
@@ -0,0 +1,234 @@
+package io.github.mivek.service;
+
+import io.github.mivek.enums.CloudQuantity;
+import io.github.mivek.model.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.ArrayList;
```

Comment:
Can you remove the unused imports ?

You can find the list of issues reported by sonar [here](https://sonarcloud.io/summary/new_code?id=io.github.mivek%3AmetarParser&pullRequest=626).

You can ignore the issue about the singleton